ranker/modules/omnicorp_overlay.py

```diff
@@ -70,7 +72,7 @@ async def add_shared_pmid_counts(
                         # If we're returning a count, then returning an empty list here is gibberish, and causes an error in weighting.
                         # {'original_attribute_name': 'publications', 'attribute_type_id': 'biolink:publications', 'value_type_id': 'EDAM:data_0006', 'value': []},
                         {
-                            "attribute_type_id": "biolink:primary_knowledge_source",  # the ‘key’*
+                            "attribute_type_id": "primary_knowledge_source",  # the ‘key’*
```

Comment:
Does this need to go into a `sources` list instead of as an attribute?

Reply:
Oops, you are correct.